### DISCUSSION TITLE
Footer layout

### DISCUSSION THREAD

**Discussion by foo4foo** [gotenberg/gotenberg#447] [Q&A]
Hello,

I'm trying to design a footer but I'm not able to figure it out. This is my `footer.html`:
```
<html>
  <head>
    <style>
      body {
        font-size: 8rem;
        margin: 4rem auto;
      }
    </style>
  </head>
  <body>
    <div style="display: flex; justify-content: space-evenly; border-top: 1px solid grey; width: 100%;">
      <p>ASDF</p>
      <p class="date"></p>
      <p><span class="pageNumber"></span> / <span class="totalPages"></span></p>
    </div>
  </body>
</html>
```

This is how it looks like when pdf is generated (endpoint used: `/forms/chromium/convert/html`):
![image](https://user-images.githubusercontent.com/16281362/163655199-5f4bf4e1-5526-4847-ace2-83e6f9fe9457.png)

Why does the top border start so far to the right and `ASDF` paragraph is not ?

I thought `display: flex` with `width: 100%` would take the whole page width.

**Comment by gulien** [comment]
No idea, Chrome handling of the header/footer feature is not well documented. Do not hesitate to look for answers using [puppeteer](https://github.com/puppeteer/puppeteer) as key word; Gotenberg uses the same API, but the `puppeteer` community is way larger. Chances are that you find an answer to your question there.

**Comment by foo4foo** [comment]
In case someone else has the same problem, I got desired result by hardcoding margins.

footer.html:
```
<html>
  <head>
    <style>
      body {
        width: 1000px;
        margin-top: 4rem;
        margin-bottom: 4rem;
        margin-left: 35px;
      }

      p {
        font-size: 8rem;
        color: lightgray;
      }
    </style>
  </head>
  <body>
    <div style="display: flex; width: 1000px;">
      <p style="margin-right: 165px;">ASDF</p>
      <p style="margin-right: 225px;" class="date"></p>
      <p><span class="pageNumber"></span> / <span class="totalPages"></span></p>
    </div>
  </body>
</html>
```
